### PR TITLE
Feature/select2 responsible copy projects

### PR DESCRIPTION
**Pull request by GarstgerUnhold**
https://www.openproject.org/issues/1966. This is the same as #533, but for the dev branch.

### CODE REVIEW

**Comment by GarstgerUnhold** [comment]
The Global Roles plugin needs to adapt to those changes, the PR can be found here: https://github.com/finnlabs/openproject-global_roles/pull/12

**Comment by hschink** [comment]
Rebased PR so Travis can have a look on it without disturbing cucumber fails.

**Comment by hschink** [comment]
The following cuke is failing constantly: `features/work_packages/work_package_show.feature:183` Please have a look on it.

File: app/assets/javascripts/autocompleter.js [hunk 1]
```diff
@@ -170,10 +170,18 @@
...
-      var data = [];
+      var data,
+          multiple = $(element).attr("multiple");
+      if (multiple !== null && multiple !== undefined && multiple) {
+        data = []
+      };
       if (!($(element).attr("data-selected") === "" || $(element).attr("data-selected") === null || $(element).attr("data-selected") === undefined)) {
         JSON.parse($(element).attr('data-selected')).each(function (elem) {
-          data.push({id: elem[1], name: elem[0]});
+          if (multiple) {
```
Comment (nkenneweg) [inline app/assets/javascripts/autocompleter.js:180]: This broke a lot of timelines select2-multiple-selects. :-1:

**Pull request merged**